src/Components/Blazor/Build/test/BindRazorIntegrationTest.cs

```diff
@@ -163,7 +163,7 @@ Task IComponent.SetParametersAsync(ParameterCollection parameters)
                 frames,
                 frame => AssertFrame.Component(frame, "Test.MyComponent", 3, 0),
                 frame => AssertFrame.Attribute(frame, "Value", 42, 1),
-                frame => AssertFrame.Attribute(frame, "OnChanged", typeof(EventCallback<UIChangeEventArgs>), 2));
+                frame => AssertFrame.Attribute(frame, "OnChanged", typeof(EventCallback<int>), 2));
```

Comment:
The original assertion was wrong here. We want this to generate strong typed stuff that won't try to do conversions. This is an update based on changes to the compiler that fixed the underlying issue.